### DISCUSSION TITLE
Programmatically changing parameters in webpage?

### DISCUSSION THREAD

**Discussion by themindfactory** [jscad/OpenJSCAD.org#1127] [Q&A]
Example might be a 3D model and on the webpage with 3 pull-down menus to adjust height length width in realtime... Looking for example.

Thanks

**Comment by hrgdavor** [comment]
I personally am not sure what the question here is :) . Please elaborate

**Reply by themindfactory** [reply]
Example:

You have a webpage with a realtime 3D model and on the same page you have 3 simple text boxes (input fields)  marked height, width, length, and as the user changes the values in these text boxes the 3D model on the page reflects the values entered in realtime...

Hope thats clear.
Thanks!

**Reply by hrgdavor** [reply]
there is not yet such example. I agree with @z3dev thyt best bet is to start with packages/utils/regl-renderer/demo.html

the rest really depends on you JS skills.

**Comment by z3dev** [comment]
@themindfactory did you have any reactive framework in mind? if not then then you can start by looking at the really simple example at packages/utils/regl-renderer/demo.html

**Comment by z3dev** [comment]
@themindfactory Theres a list of web sites on the the User Guide. Hopefully that provides a few nice examples.

https://openjscad.xyz/dokuwiki/doku.php?id=en:user_guide_help
